Java support/java-no-code-sample/src/main/java/com/github/kelseyhightower/nocode/JavaNoCodeSampleApp.java

```diff
@@ -0,0 +1,13 @@
+package com.github.kelseyhightower.nocode;
+
+/**
+ * A sample app that shows how to integrate Java with No Code.
+ */
+public class JavaNoCodeSampleApp {
```

Comment:
Excuse me but you broke the rule.

Reply:
@ajwhite thank you for pointing out my mistake. I have a different solution that should be acceptable.